percona/controllers/pmm/pmm.go

```diff
@@ -0,0 +1,266 @@
+package pmm
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+
+	"github.com/percona/percona-postgresql-operator/internal/kubeapi"
+	crv1 "github.com/percona/percona-postgresql-operator/pkg/apis/crunchydata.com/v1"
+
+	"github.com/pkg/errors"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	pmmContainerName = "pmm-client"
+)
+
+func UpdatePMMSidecar(clientset kubeapi.Interface, cluster *crv1.Pgcluster, deployment *appsv1.Deployment) error {
+	ctx := context.TODO()
+	cl, err := clientset.CrunchydataV1().PerconaPGClusters(cluster.Namespace).Get(ctx, cluster.Name, metav1.GetOptions{})
+	if err != nil {
+		return errors.Wrap(err, "get perconapgcluster resource: %s")
+	}
+
```

Comment:
checking if `cl.Spec.PMM.Enabled` there will be more clear instead inside `AddPMMSidecar` function

Reply:
we need to remove pmm container if `cl.Spec.PMM.Enabled=false`

Reply:
then it misslead more, when `AddPMMSidecar` function used to remove it =)

Reply:
yeah, you are right, will rename it)